CODE/Logicytics.py

```
@@ -286,69 +288,93 @@
+            EXECUTION_LIST = generate_execution_list()
+            for file in range(len(EXECUTION_LIST)):  # Loop through List
+                log.parse_execution(Execute.script(EXECUTION_LIST[file]))
+                log.info(f"{EXECUTION_LIST[file]} executed")
         except UnicodeDecodeError as e:
             log.error(f"Error in code: {e}")
         except Exception as e:
             log.error(f"Error in code: {e}")
 
 
-@log.function
 def zip_generated_files():
```

Ruff (0.8.0)

Missing return type annotation for public function `zip_generated_files`

Add return type annotation: `None`

(ANN201)